editor.html

```diff
@@ -337,6 +337,13 @@ <h2>Flashing micro:bit</h2>
         <script src="static/js/microbit-fs.umd.js"></script>
         <script src="micropythonapi.js"></script>
         <script src="python-main.js"></script>
+	    <script src="lzma/lzma_worker.js"></script>
+	    <script src="base64-js/base64.js"></script>
+        <script src="hexlify.js"
+            type="application/javascript"></script>
+        <script src="urlparser.js"></script>
+        <script src="python-main.js"
+            type="application/javascript"></script>
```

Comment:
This looks like a regresion from the rebase, this PR should only contain the `<script src="urlparser.js"></script>` addition.

Reply:
Done.

Reply:
Done